include/wx/gtk/mimetype.h

```diff
@@ -23,7 +23,11 @@
 class WXDLLIMPEXP_CORE wxGTKMimeTypesManagerImpl : public wxMimeTypesManagerImpl
 {
 protected:
+#if defined(__UNIX__)
     wxString GetIconFromMimeType(const wxString& mime) wxOVERRIDE;
+#else
+    wxString GetIconFromMimeType(const wxString& mime);
```

Comment:
@vadz I have committed the changes to only use  GetIconFromMimeType under Unix.

Tim S.